docs/guides/headless-cms/build-static-website/next-13.md

```diff
@@ -137,8 +137,8 @@ export default async function DynamicPage({ params }) {
 ```
 
 Go to `http://localhost:3000/about`, replacing `about` with any of your item slugs. Using the Directus JavaScript SDK,
-the single item with that slug is retrieved, and the page should show your data. `readOne()` only checks against your
-`slug` Primary ID Field.
+the single item with that slug is retrieved, and the page should show your data. `readItem()` allows you to specify the
+`slug` or Primary ID Field.
```

Comment:
```suggestion
the single item with that slug is retrieved, and the page should show your data. `readItem()` allows you to specify the Primary ID Field.
```

Lets drop the `slug` here because that may cause people to make false assumptions. Only the primary keys works (and the slug only if that happens to be the primary key)